content/swift/concepts/dictionaries/terms/contains/contains.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: '.contains'
+Description: 'a Boolean value that indicates whether a dictionary contains a specified key-value pair.'
+Subjects:
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'Dictionaries'
+  - 'Properties'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.contains`** method returns a Boolean value that indicates whether a dictionary contains a specified key-value pair. The method takes a single argument, which is a tuple that contains a key-value pair. If the dictionary contains the specified key-value pair, the method returns `true`; otherwise, it returns `false`.
+
+## Syntax
+
+```pseudo
+dictionaryInstance.contains(where: { key, value in
+  return key == "key" && value == "value"
+})
+```
+
+## Example
+
+```swift
+var fruitStand = [
+  "Apples": 12,
+  "Bananas": 20,
+  "Oranges": 17
+] 
+
+print(fruitStand.contains(where: { key, value in
+  return key == "Apples" && value == 12
+}))
+// Output: true 
+
+print(fruitStand.contains(where: { key, value in
+  return key == "Grapes" && value == 10
+}))
+// Output: false 
+```
+
+In the example above, the `fruitStand` dictionary contains key-value pairs for apples, bananas, and oranges. The first `print` statement checks if the dictionary contains the key-value pair "Apples": 12, which is true. The second `print` statement checks for the key-value pair "Grapes": 10, which is false.
+
+The `.contains` method is useful for checking the presence of specific key-value pairs in a dictionary.

```

Comment:
```suggestion
The `.contains` method is useful for checking the presence of specific key-value pairs in a dictionary.

```
It typically checks for the existence of the key,  not for key-value pairs. Please make this correction.